python/python/tests/test_transaction.py

```diff
@@ -230,18 +238,18 @@ async def transaction_test(
     args.append(1)
 
     transaction.geoadd(
-        key9,
+        key13,
```

Comment:
why is this here?

Reply:
before these changes, the geo commands here were tested using key9 which was originally used for a list that was empty after line 178. I could make key13 a list key as well and use that to test blpop/brpop but I thought it made more sense to use key9 for only lists and use key13 as a dedicated geo key

Reply:
why key13 instead key12?

Reply:
I believe key12 is being added in another PR, was trying to prevent merge conflicts, let me know if you want me to change to key12